cmd/syncthing/gui.go

```diff
@@ -997,37 +997,22 @@ func (s *apiService) postDBIgnores(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *apiService) getEvents(w http.ResponseWriter, r *http.Request) {
```

Comment:
Ah, no, this is not what I meant, sorry. :) It would be cleaner to have the two handlers as you did, but they both call out to a third method that does most of the work, with the specific event subscription to return as a parameter.